ocs_ci/deployment/provider_client/storage_client_deployment.py

```diff
@@ -224,7 +224,10 @@ def provider_and_native_client_installation(
                     "replica"
                 ] = no_of_worker_nodes
 
-                if self.platform in constants.HCI_PROVIDER_CLIENT_PLATFORMS:
+                if self.platform in [
```

Comment:
we need to use HCI_BAREMETAL until it is renamed on the decided new platform name, or until we find all dependencies and replace it with just "baremetal" all over the places